src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -335,8 +368,9 @@ void buildHearingPartiesPartyDetailsAppointee(String isAppointee, boolean expect
             null,
             hearingOptions,
             HearingSubtype.builder().build(),
-            PARTY_ID,
-            null, referenceData
+            null,
+            referenceData,
+            null
         );
 
         assertThat(partiesDetails.stream().filter(o -> PARTY_ID.substring(0,15).equalsIgnoreCase(o.getPartyID())).findFirst()).isPresent();
```

Comment:
I see that this stream pattern is throughout the test class. Perhaps you can find a way to use extracting and replace all of them?